CMakeLists.txt

```diff
@@ -126,7 +126,8 @@ if(BRLCAD_FOUND)
   message(STATUS "BRL-CAD found.")
   message(STATUS "BRLCAD_LIBRARIES: ${BRLCAD_LIBRARIES}")
   message(STATUS "BRLCAD_INCLUDE_DIRS: ${BRLCAD_INCLUDE_DIRS}")
-  # Include the BRL-CAD include directories
+  # Include the BRL-CAD include 
+  target_compile_definitions(cadventory PRIVATE RT_DLL_IMPORTS)
```

Comment:
This belonged in another commit so you could document the rationale/need as it has nothing to do with the commit message.